app/controllers/provider/admin/account/email_configurations_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Provider::Admin::Account::EmailConfigurationsController < Provider::Admin::Account::BaseController
+
+  # We only enable it for master for now, feel free to roll it out for any provider
+  before_action :ensure_master_domain, :email_configurations_enabled?
```

Comment:
Could we inherit from https://github.com/3scale/porta/blob/25fa9bc53bdd3eeb8f2a9c8e882f771c85c3c3cb/app/controllers/provider/base_controller.rb instead that includes this already?

Reply:
Actually, if this will be available for providers in the future, we don't want to inherit from that. It's fine like this, explicit, so that we can remove it when we make it available for all providers (instead of changing the parent class again). WDYT?